scylla/README.md

```diff
@@ -0,0 +1,53 @@
+# Agent Check: Scylla
+
+## Overview
+
+This check monitors [Scylla][1] through the Datadog Agent.
```

Comment:
This Datadog-[Scylla][1] integration collects a majority of the exposed metrics by default, with the ability to customize additional groups based on specific user needs.